packages/connect/CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
-|             Package              | Stable |    Canary    |
-| :------------------------------: | :----: | :----------: |
-|       npm @trezor/connect        | 9.2.4  | 9.2.5-beta.2 |
-|     npm @trezor/connect-web      | 9.2.4  | 9.2.5-beta.2 |
-| npm @trezor/connect-webextension | 9.2.4  | 9.2.5-beta.2 |
-
-|     Deployment     | Stable |    Canary    |
-| :----------------: | :----: | :----------: |
-| connect.trezor.io/ | 9.2.4  | 9.2.5-beta.2 |
+|             Package              | Stable | Canary |
+| :------------------------------: | :----: | :----: |
+|       npm @trezor/connect        | 9.3.0  |   -    |
+|     npm @trezor/connect-web      | 9.3.0  |   -    |
+| npm @trezor/connect-webextension | 9.3.0  |   -    |
+
+|     Deployment     | Stable | Canary |
+| :----------------: | :----: | :----: |
+| connect.trezor.io/ | 9.3.0  |   -    |
```

Comment:
so aren't we skipping `9.2.5` version here? I remember there was this discussion going on but I probably need to go through it again.

Reply:
Yes, we are modifying the files we publish in `@trezor/connect-webextension` and we agreed on always doing a minor release when we do that. Of course we can discuss it again. https://github.com/trezor/trezor-suite/pull/12761#issuecomment-2154548777